libs/sdk-ui/src/base/localization/TranslationsCustomizationProvider/workspaceSettingsLoader.ts

```diff
@@ -15,7 +15,7 @@ class WorkspaceSettingsLoader {
             settings = backend
```

Comment:
Please don't forget to update the copyright accordingly.

Reply:
fixed